<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{ 
        background-image: url(&quot;src/assets/background.jpg&quot;);
        background-repeat: no-repeat; 
        background-position: center;}
    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="verticallayout_manual">
       <property name="styleSheet">
        <string notr="true">#verticallayout_manual {border-width: 1px;
 border-style: solid;
 border-color: black;
 border-radius:10px;
 padding:10px 5px;}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_manual">
        <item>
         <layout class="QVBoxLayout" name="manula_path">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="manual_file_path">
            <item>
             <widget class="QPushButton" name="manual_file_path_button">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #2B5DD1; color: #FFFFFF ; border-style: outset; 
padding: 8px 0px ;  ; border-radius: 10px ; border-color: #2752B8;</string>
              </property>
              <property name="text">
               <string>file path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="manual_file_path_text">
              <property name="text">
               <string>select file</string>
              </property>
              <property name="margin">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="manual_result_path">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="manual_result_path_button">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #2B5DD1; color: #FFFFFF ; border-style: outset; 
padding: 8px 0px ;  ; border-radius: 10px ; border-color: #2752B8;</string>
              </property>
              <property name="text">
               <string>result path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="manual_result_path_text">
              <property name="text">
               <string>select folder</string>
              </property>
              <property name="margin">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="manual_start">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FF6987; color: #FFFFFF ; border-style: outset; 
padding: 8px 40px ;  ; border-radius: 10px ; border-color: #2752B8;</string>
          </property>
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="logo_home">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(&quot;src/assets/home.jpg&quot;);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logo_history">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(&quot;src/assets/home.jpg&quot;);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="logo_email">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(&quot;src/assets/home.jpg&quot;);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logo_help">
         <property name="maximumSize">
          <size>
           <width>42</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(&quot;src/assets/home.jpg&quot;);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logo_back">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(&quot;src/assets/home.jpg&quot;);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets/test.qrc"/>
 </resources>
 <connections/>
</ui>
